application/positionmanager/src/positionmanager/clients.py

```
@@ -71,30 +72,34 @@ def get_data(
-            response = requests.post(endpoint, json=date_range.to_payload(), timeout=10)
+            response = requests.get(endpoint, params=params, timeout=30)
         except requests.RequestException as err:
             raise RuntimeError(f"Data manager service call error: {err}") from err
 
-        if response.status_code != 200:
+        if response.status_code == 404:
+            return pl.DataFrame()
+        elif response.status_code != 200:
             raise Exception(
                 f"Data service error: {response.text}, status code: {response.status_code}",
             )
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)